docs/rst/notes/previous_versions/v0.9.0.rst

```diff
@@ -0,0 +1,12 @@
+Version 0.9.0
```

Comment:
Please, update old `previous_versions` files so all have the same title subline (e.g. all with `======`).
Otherwise it is not correctly indexed

Reply:
applied in https://github.com/eProsima/Fast-DDS-statistics-backend/pull/193/commits/2036c5afc44e7901ef22927ddf007869177f1eef